Riva0110/Vocab-Garden
# Feature/mobile version

### Description

* Add RWD
* Add feature of word selection for mobile version

### Test Plan

* Check if all features still work 
* Check if user experience is smooth
